xla/service/gpu/runtime/BUILD

```diff
@@ -25,11 +24,25 @@ package_group(
     ],
 )
 
+cc_library(
+    name = "gpu_kernel_helper",
+    hdrs = if_gpu_is_configured(["gpu_kernel_helper.h"]),
```

Comment:
It looks to me that this header includes cuda code? In that case, this would have to be a cuda_library for the if_cuda_is_configured case.

Reply:
Yes, this should be fine, because gpu_kernel_helper is "header-only" library, i.e., it's not really compiled, 
only provides macros/typedefs/templates for other CUDA/ROCM libraries

Reply:
> Yes, this should be fine, because gpu_kernel_helper is "header-only" library, i.e., it's not really compiled, only provides macros/typedefs/templates for other CUDA/ROCM libraries

unfortunately header-only doesn't help, internally we also compile these header-only targets.

Reply:
OK, I see, I changed gpu_kernel_helper to be compiled as gpu library